libcodechecker/log/build_action.py

```diff
@@ -57,6 +57,14 @@ def __setattr__(self, attr, value):
 
     def __eq__(self, other):
         return other.original_command == self.original_command
+    
+    def to_dict(self):
```

Comment:
This is one format,  when we want to serialize it, shouldn't we support the newer compile command json where the command is a list of arguments?

Will be the space characters in compilation arguments or file names escaped, quoted right when we write this out?

Reply:
this time we will support the old format only. When we transform to the new format later, the unique_compile_commands.json shall follow that new format too.